### PR TITLE
Update extract frames guide

### PR DESCRIPTION
**Pull request by sfmig**
- update paths in extract frames guide
- some small edits in labelling guide (I got distracted...)
- remove dev version of labelling guide

### CODE REVIEW

File: guides/ExtractFramesCluster.md [hunk 1]
```diff
@@ -70,23 +72,26 @@
    sacct -X -j 3813494,3813184
    ```
 
+   To check the timelimit of the jobs submitted by user `sminano`
```
Comment (samcunliffe) [inline guides/ExtractFramesCluster.md:75]:
```suggestion
   To check the time limit of the jobs submitted by a user (for example, `sminano`)
```

File: guides/ExtractFramesCluster.md [hunk 2]
```diff
@@ -70,23 +72,26 @@
...
+
 ---
 
 ### Notes:
 
-- For a new bash script, it is often a good idea to try each of the steps with an interactive node first!
+- When developing a new bash script, it is often a good idea to try each of the steps in an interactive node first!
 
-  To request a GPU one:
+  To request a GPU interactive node:
```
Comment (samcunliffe) [inline guides/ExtractFramesCluster.md:93]:
```suggestion
  To request an interactive GPU node:
```

File: guides/ManualLabellingSteps.md [hunk 3]
```diff
@@ -104,17 +98,22 @@ If we are launching the tool for the first time for a new project:
...
+  - I found this less useful
+
 - Some convenient shortcuts:
 
   - Press `b` to toggle the boxes visibility
   - Press `l` to toggle the visibility of the boxes' IDs
   - Press `spacebar` to show the annotations for that frame
 
-- There is no undo action in the annotation tool!
+- **Important**: There is no undo action in the annotation tool!
```
Comment (samcunliffe) [inline guides/ManualLabellingSteps.md:111]:
```suggestion
> [!CAUTION]
> There is no undo action in the annotation tool!
```

[Assuming this is only rendered in GFM](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts).

**Review by samcunliffe — APPROVED** [review]
> some small edits in labelling guide (I got distracted...)

It's more of an update to the labelling guide.

File: guides/ManualLabellingSteps.md (hunk 3 above)
Comment (sfmig) [reply guides/ManualLabellingSteps.md:111]: yes I was looking for this, thanks

**Pull request merged**